shared/login/login/index.desktop.js

```diff
@@ -1,11 +1,58 @@
 // @flow
 import React, {Component} from 'react'
-import {Box, UserCard, Text, Button, FormWithCheckbox, Dropdown} from '../../common-adapters'
-import {globalStyles, globalColors} from '../../styles'
+import {Box, UserCard, Text, Button, FormWithCheckbox, Icon, PopupDialog} from '../../common-adapters'
+import {globalStyles, globalColors, globalMargins} from '../../styles'
+import glamorous from 'glamorous'
 
 import type {Props} from '.'
 
-class Login extends Component<void, Props, void> {
+type State = {
+  open: boolean,
+}
+
+const ItemBox = glamorous(Box)({
+  ...globalStyles.flexBoxCenter,
+  ':hover': {
+    backgroundColor: globalColors.blue3_40,
+  },
+  border: `1px solid ${globalColors.lightGrey2}`,
+  minHeight: 40,
+  width: '100%',
+})
+
+const ButtonBox = glamorous(Box)({
+  ...globalStyles.flexBoxRow,
+  ':hover': {
+    border: `solid 1px ${globalColors.blue2}`,
+    color: globalColors.blue2,
+  },
+  alignItems: 'center',
+  color: globalColors.lightGrey2,
+  border: `solid 1px ${globalColors.lightGrey2}`,
+  borderRadius: 100,
+  paddingRight: globalMargins.small,
+  width: 270,
+})
+
+const other = 'Someone else...'
+
+const UserRow = ({user, onClick}) => (
```

Comment:
simple replacement for dropdown. this whole screen should be redone, its very old